### PR TITLE
Add baner to the login page by MARIONEQ

### PR DESCRIPTION
**Pull request by Zaptyp**
Add baner to the login page by MARIONEQ

### CODE REVIEW

File: frontend/src/components/Login/Baner.vue [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        <v-col>
+           <v-row align="center">
+              <v-card dark color="#d32f2f" style="height: 524px; margin-right: 35px; width: 350px;">
+                <v-card-text>
+                  <div style="width: 140; height: 140;">
+                    <img src="./iconTwo.svg"
+                    width="140"
+                    class="logo"/>
+                  </div>
+                <div id="hedlineRed">Wulkanowy</div>
```
Comment (MarcinK50) [inline frontend/src/components/Login/Baner.vue:12]: hedline?

File: frontend/src/assets/wulkanowy.svg [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+5<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (Pengwius) [inline frontend/src/assets/wulkanowy.svg:1]: What this 5 means?

File: frontend/src/components/Login/Baner.vue [hunk 3]
```diff
@@ -0,0 +1,56 @@
+<template>
+    <div class="card" style="float: left;">
+        <v-col>
+           <v-row align="center">
+              <v-card dark color="#d32f2f" style="height: 524px; margin-right: 35px; width: 350px;">
+                <v-card-text>
+                  <div style="width: 140; height: 140;">
+                    <img src="./iconTwo.svg"
```
Comment (Pengwius) [inline frontend/src/components/Login/Baner.vue:8]: What is iconTwo? Could you use descriptive name?

**Review by MarcinK50 — CHANGES_REQUESTED** [review]
change the hedline and rename PP.vue to something other

**Review by Pengwius — CHANGES_REQUESTED** [review]

**Review by Pengwius — APPROVED** [review]

**Review by MarcinK50 — APPROVED** [review]
good

**Pull request merged**